tests/config.c

```diff
@@ -155,7 +156,7 @@ config_parse(master_config *cfg, const uint8 num_config, int argc, char *argv[])
          }
          config_set_uint64("extent-size", cfg, extent_size)
          {
-            for (cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
+            for (int cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
```

Comment:
I see what you mean by the above comment.

All these `int cfg_idx` should have been `int8` when I made this bug-fix, to match what was on old line L131.

I changed it accordingly to be `uint8`.

And, oh, btw, this variable `num_config` is misleading; it is really supposed to be `num_tables` (for which we are setting up these config structs). And num-tables is limited elsewhere to 8, and other code is using `uint8` for that variable's type.